.github/workflows/release.yml

```
@@ -8,51 +8,22 @@ jobs:
-      - name: Package
-        run: |
-          mkdir dist
-          cp main.js manifest.json README.md dist/
-          zip -r release.zip dist/
-
-      - uses: actions/create-release@v1
-        id: create_release
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Create Release and Upload Assets
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)